src/unix/linux_like/linux/musl/mod.rs

```diff
@@ -344,6 +344,73 @@ s! {
         pub len: ::__u32,
         pub options: ::__u32,
     }
+
+    // netinet/tcp.h
+
+    pub struct tcp_info {
+        pub tcpi_state: u8,
+        pub tcpi_ca_state: u8,
+        pub tcpi_retransmits: u8,
+        pub tcpi_probes: u8,
+        pub tcpi_backoff: u8,
+        pub tcpi_options: u8,
+        /*
+         * FIXME(musl): when musl headers are more up to date
+        // bitfields 4: tcp_snd_wscale, tcp_rcv_wscale
+        pub tcpi_snd_rcv_wscale: u8,
+        // bitfields 1
+        pub tcpi_delivery_rate_app_limited: u8,
+        // bitfields 2
+        pub tcpi_fastopen_client_fail: u8,
```

Comment:
I think this needs the bitfield change too

```suggestion
        /// This contains the bitfields `tcpi_delivery_rate_app_limited` (one bit) and
        /// `tcpi_fastopen_client_fail` (two bits).
        pub tcpi_delivery_fastopen_bitfields: u8,
```